src/main/resources/assets/paths/approveDataset.yaml

```diff
@@ -0,0 +1,49 @@
+put:
+  summary: DAC Chair updates dataset approval status
+  description: Endpoint allows DAC Chairs to submit approval decisions for datasets under their purview (or requested purview)
+  tags:
+    - DAC
+  parameters: 
+    - name: dacId
+      in: path
+      required: true
+      schema:
+        type: integer
+    - name: datasetId
+      in: path
+      required: true
+      schema:
+        type: integer
+  requestBody:
+      description: 'Payload that should contain the key-value pair of {"approval": Boolean}'
```

Comment:
minor doc nit:
```suggestion
      description: 'Payload that should contain the key-value pair of {"approval": true|false}'
```

Reply:
Alternatively, a new model class could be created for `DatasetApproval` with a single field, `approval`.